src/plugins/azgenerator/azgenerator.ts

```diff
@@ -17,6 +18,32 @@ export async function processRequest(host: Host) {
             model.CliCoreLib = cliCoreLib;
         }
 
+        // Read the argument of generation-mode, detect if needed
+        let generationMode = await host.GetValue('generation-mode') || "auto";
```

Comment:
Doesn't it suppose to be either full or incremental ?

Reply:
Default should be full right?

Reply:
Default is auto